PIL/Image.py

```diff
@@ -2206,6 +2206,8 @@ def fromqpixmap(im):
     # first two members of shape are set to one
     # ((1, 1), "|b1"): ("1", "1"), # broken
     ((1, 1), "|u1"): ("L", "L"),
+    ((1, 1), "<u2"): ("L", "I;16"),
+    ((1, 1), ">u2"): ("L", "I;16B"),
     ((1, 1), "|i1"): ("I", "I;8"),
```

Comment:
These two unpackers are only taking the upper byte and throwing away the lower byte, and packing this into a single channel 8 bit image.  In the process, they're interpreting the upper byte as a uint8, which is unlikely to be correct if it's a signed int type. 

I think that the correct mode, raw_mod tuples to use would be `('I', 'I;16S')`  and `('I', 'I;16BS')`. These two promote the image to a 32 bit image, which is enough range to losslessly contain the I16S datatype.